File: URI Programs/C/1048.c
#include <stdio.h>

int main(){
	float salary, earned;
	int percent;
	scanf("%f", &salary);
	if(salary <= 400){
		earned = salary * 0.15;
		salary = salary * 1.15;
		percent = 15;
	} else if (salary <= 800){
		earned = salary * 0.12;
		salary = salary * 1.12;
		percent = 12;
	} else if (salary <= 1200){
		earned = salary * 0.1;
		salary = salary * 1.1;
		percent = 10;
	} else if (salary <= 2000){
		earned = salary * 0.07;
		salary = salary * 1.07;
		percent = 7;
	} else {
		earned = salary * 0.04;
		salary = salary * 1.04;
		percent = 4;
	}
	printf("Novo salario: %.2f\n", salary);
	printf("Reajuste ganho: %.2f\n", earned);
	printf("Em percentual: %d %\n", percent);
	return 0;
}

File: URI Programs/Python/1048.py
x = float(input())

if x < 400.00:
	p = 15
	new = x + (x*0.15)
elif x < 800.00:
	p = 12
	new = x + (x*0.12)
elif x < 1200.00:
	p = 10
	new = x + (x*0.10)
elif x < 2000.00:
	p = 7
	new = x + (x*0.07)
else:
	p = 4
	new = x + (x*0.04)
print("Novo salario: " + str("%.2f" %new) + "\n" + "Reajuste ganho: " + str("%.2f" (x*p/100)) + "\n" + "Em percentual: " + str(p))